### PR TITLE
Updated my caption

### PR DESCRIPTION
**Pull request by tmahesh1975**

### CODE REVIEW

**Review by jiny-joseph — APPROVED** [review]
Good job

File: _slides/15-tmahesh1975.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Updated my Caption to Fun caption
```
Comment (shrutim97) [inline _slides/15-tmahesh1975.md:6]: great

**Review by shrutim97 — APPROVED** [review]
review

**Comment by tmahesh1975** [comment]
All looks ok

**Pull request closed without merging**